samples/bootloader/README.rst

```diff
@@ -86,37 +70,39 @@ The sample accomplishes a simple and reliable *Root of Trust* (RoT) for a secure
 Provisioning
 ============
 
-The public key hashes are not compiled with the source code of the bootloader sample.
-Instead, they must be stored in a separate memory region through a process called *provisioning*.
+The public key hashes are not compiled with the source code of the NSIB.
+Instead, they must be written to the device in a process called *provisioning*.
+
+The hashes are automatically generated by the build system based on the specified private key and the additional public keys.
 
-By default, the bootloader sample automatically generates and provisions public key hashes directly into the bootloader HEX file, based on the specified private key and the additional public keys.
+By default, the hashes are placed directly into the NSIB HEX file and then automatically provisioned when the HEX file is programmed to the device.
 
-Alternatively, to facilitate the manufacturing process of a device using the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
-If you choose to do so, use the Python scripts located in the :file:`scripts/bootloader` folder to create and provision the keys manually.
+Alternatively, in a more realistic manufacturing process, the NSIB HEX file and the HEX file containing the hashes can be programmed separately.
+For this approach, use the Python scripts located in the :file:`scripts/bootloader` folder.
 
-In both cases, the bootloader accesses the provisioned data using the :ref:`doc_bl_storage` library.
+In either case, the NSIB accesses the provisioned data at runtime using the :ref:`doc_bl_storage` library.
 
 .. _bootloader_provisioning_otp:
 
-OTP Regions
+OTP regions
 -----------
 
-On some SoCs/SiPs, such as the nRF9160 or nRF5340, the provisioned data is held in the *one-time programmable* (OTP) region in the *User Information Configuration Registers* (UICR).
+For the nRF SoCs that support it, such as the nRF9160 or nRF5340, the provisioned data is held in the *one-time programmable* (OTP) region in the *User Information Configuration Registers* (UICR) instead of the internal flash memory.
 OTP regions are special regions of the flash memory that only allow flash memory writes in half-word lengths, and *only* when the target half-word is the value ``0xFFFF``.
 
 Because of these design constraints, the following limitations apply:
 
-* The public key hash cannot contain half-words with the value ``0xFFFF``, as such hashes cannot be guaranteed to be immutable when placed in the OTP region.
-  Therefore, the bootloader refuses to boot if any hash contains a half-word with the value ``0xFFFF``.
-  If your public key hash is found to contain this value, :ref:`regenerate it <ug_fw_update_keys>` or use another public key.
+* The public key hash may not contain half-words with the value ``0xFFFF``, as such hashes cannot be guaranteed to be immutable when placed in the OTP region.
```

Comment:
```suggestion
* The public key hash can not contain half-words with the value ``0xFFFF``, as such hashes cannot be guaranteed to be immutable when placed in the OTP region.
```

Reply:
This is a limitation that we are asking the user to respect. Is may then not more appropriate than can ?

Reply:
Ok then I'd go with must then. I mean, there is no scenario in which they can use a half-word with vaue 0xFFFF, right?

Reply:
^this

Reply:
correct